lib/ansible/modules/network/fortimanager/fmgr_query.py

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community"
+}
+
+DOCUMENTATION = '''
+---
+module: fmgr_query
+version_added: "2.8"
+author: Luke Weighall - lweighall
```

Comment:
Please change the author doc to be consistent with Ansible modules
```suggestion
author: Luke Weighall (@lweighall)
```

Reply:
We had some approved, and the @ mentions caused some kind of cascading notification. We literally got an email from Ansibot itself, to remove these. 

Please sort out the actual standard and let us know.

Reply:
@ftntcorecse Is the message you received:

> This PR contains `@` mentions in at least one commit message. Those mentions can cause cascading notifications through GitHub and need to be removed. Please squash or amend your commits to remove the mentions.

That is about **commit messages**, rather than the contents of any file updated by this PR.

If you have a suggestion on how we can improve the wording I'll happily update https://github.com/ansible/ansibullbot/blob/master/templates/commit_msg_mentions.j2

The correct format is:
```
author: Luke Weighall (@lweighall)
```
or
```
author:
  - Luke Weighall (@lweighall)
  - John Barker (@gundalow)
```

This format allows Ansibulbot to `@mention` the correct set of people when future Issues or PRs are raised. https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields